deploy/test-external-secrets-watch-deployment.yaml

```diff
@@ -90,8 +90,8 @@ spec:
     purgeUnmanagedConfig:
       enabled: true
       exclude:
-        # Auth is not implemented yet, but just for demonstration.
-        auth: true
+        # secretsEngines is not implemented yet, but just for demonstration.
+        secretsEngines: true
```

Comment:
@wazery as you suggested let's rename this as `secrets`.

Reply:
Done